snippets/swift-testing.code-snippets

```diff
@@ -0,0 +1,53 @@
+{
+  "@Test": {
+    "prefix": "@Test",
+    "body": [
+      "@Test func ${1:name}() async throws {",
+      "\t$2",
+      "}"
+    ],
+    "description": "swift-testing: Test Function"
+  },
+  "@Test(\"named\")": {
+    "prefix": "@Test",
+    "body": [
+      "@Test(\"${1:display name}\")",
+      "func ${2:name}() async throws {",
+      "\t$3",
+      "}"
+    ],
+    "description": "swift-testing: Test Function with Traits"
+  },
+  "@Suite": {
+    "prefix": "@Suite",
```

Comment:
This one is a little different because the snippet body doesn't begin with `@`—it doesn't have the `@Suite` attribute explicitly, unlike the "suite _with traits_" snippet which does. Should its prefix be simpler like just "suite"?

Reply:
On the one hand if they both has the prefix `@Suite` you can start typing `@Suite` and then use the arrow keys to pick whether you want the named or unnamed version. But on the other hand you're right, you're not actually annotating with @Suite in the unnamed one. I'll try out both and see what feels better.

Reply:
I've renamed the snippet name to be Suite, but kept the prefix as `@Suite`. This lets you easily pick between the named/unnamed with you type `@Suite` and still have the unnamed version come up in the list when you type `suite`